### PR TITLE
fix example request urls; update meta titles; remove invalid fields; …

### PR DESCRIPTION
**Pull request by spencerhunter**
…update nested tables; add transfer statuses; copy updates

### CODE REVIEW

File: pages/api-reference/connect/funding-sources/list-for-external-party.mdx [hunk 1]
```diff
@@ -5,13 +5,13 @@ title: List funding sources for an external party
...
-  title: List funding sources for an External Party | Dwolla Connect | Dwolla API Reference
-  description: Endpoint for listing funding sources for external parties.
+  title: "Dwolla API Reference | Funding Sources: List funding sources for an External Party"
+  description: "Endpoint for listing funding sources for external parties."
 ---
 
 # List funding-sources for an External Party
 
-Retrieve a list of funding sources that belong to a <span><ContentTooltip preset="external-party">external party</ContentTooltip></span>. By default, all funding sources are returned unless the removed query string parameter is set to false in the request.
+Retrieve a list of funding sources that belong to an External Party. By default, all funding sources are returned in the list and can be identified as active or removed via the `removed` attribute.
```
Comment (ShreyaThapa) [inline pages/api-reference/connect/funding-sources/list-for-external-party.mdx:14]:
```suggestion
Retrieve a list of funding sources that belong to an <span><ContentTooltip preset="external-party">External Party</ContentTooltip></span>. By default, all funding sources are returned in the list and can be identified as active or removed via the `removed` attribute.
```

Comment (ShreyaThapa) [reply pages/api-reference/connect/funding-sources/list-for-external-party.mdx:14]: Added the `external-party` tooltip. Feel free to disregard this suggestion if it doesn't fit here! :)

File: pages/api-reference/connect/transfers/index.mdx [hunk 2]
```diff
@@ -18,6 +18,18 @@ import Transfer from "../../../../assets/api-resource-tables/transfers/Transfer"
...
 
+### Transfer Statuses
+
+Once a transfer is successfully created, it can go through various status changes as it moves from Dwolla to your bank for processing. Reference the list below that includes every possible status and a brief description:
+
+| Status    | Description                                                                                                                        |
+| --------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| created   | The initial status of a transfer.                                                                                                  |
+| pending   | A transfer will move from created to pending as it processes in Dwolla system and continues processing on submission to your bank. |
```
Comment (ShreyaThapa) [inline pages/api-reference/connect/transfers/index.mdx:28]:
```suggestion
| pending   | A transfer will move from created to pending as it processes in the Dwolla system and continues processing on submission to your bank. |
```

**Review by ShreyaThapa — COMMENTED** [review]
Added some suggestions and a commit! After those are reviewed, this should be gtg! 😃

**Review by ShreyaThapa — APPROVED** [review]

**Review by lmshepherd — APPROVED** [review]

**Pull request merged**